src/current/_includes/v22.1/misc/enterprise-features.md

```diff
@@ -1,10 +1,19 @@
+### Cluster optimization
+
+[Follower Reads](follower-reads.html) | Reduce read latency in multi-region deployments by using the closest replica at the expense of reading slightly historical data.
+[Multi-Region Capabilities](multiregion-overview.html) | Row-level controls where your data is stored to help you reduce read and write latency and meet regulatory requirements.
+[Node Map](enable-node-map.html) | Visualize the geographical distribution of a cluster by plotting its node localities on a world map.
+
+### Restoration and streaming
```

Comment:
We don't really ever refer to "restores" as "restoration" in CRL docs.

I think perhaps "Backups, restores, and streaming" 

Or, "Recovery and streaming" might work.